pkgs/tools/filesystems/sasquatch/default.nix

```diff
@@ -1,34 +1,43 @@
-{ fetchFromGitHub
+{ lib
+, stdenv
+, fetchFromGitHub
 , fetchurl
-, lz4 ? null
-, lz4Support ? false
-, lzo
-, lib, stdenv
 , xz
+, lzo
 , zlib
+, zstd
+, lz4
+, lz4Support ? false
 }:
 
-assert lz4Support -> (lz4 != null);
-
 let
-  patch = fetchFromGitHub {
-    owner = "devttys0";
-    repo = "sasquatch";
-    rev = "3e0cc40fc6dbe32bd3a5e6c553b3320d5d91ceed";
-    sha256 = "19lhndjv7v9w6nmszry63zh5rqii9v7wvsbpc2n6q606hyz955g2";
-  } + "/patches/patch0.txt";
+  patch = fetchFromGitHub
+    {
+      # NOTE: This uses my personal fork for now, until
+      # https://github.com/devttys0/sasquatch/pull/40 is merged.
```

Comment:
For posterity sake I also replicated it here: https://github.com/DeterminateSystems/sasquatch